extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -584,6 +586,12 @@ export class AmpA4A extends AMP.BaseElement {
           if (method && !isEnumValue(XORIGIN_MODE, method)) {
             dev().error('AMP-A4A', `cross-origin render mode header ${method}`);
           }
+          const safeframeVersion =
+            fetchResponse.headers.get(SAFEFRAME_VERSION_HEADER);
+          if (safeframeVersion && safeframeVersion != SAFEFRAME_VERSION) {
+            this.safeframeVersion_ = safeframeVersion;
```

Comment:
Functionally, code looks great. But variable names are a bit unclear. I'd suggest renaming:
SAFEFRAME_VERSION -> DEFAULT_SAFEFRAME_VERSION (or similar prefix to indicate this is a just a starting point)
const safeframeVersion -> safeframeVersionRequestedByHeader (or similar but less wordy)
this.safeframeVersion -> leave as-is